applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -319,6 +319,17 @@ build_bridge(#state{endpoints=Endpoints
             ])
           ,wh_json:get_value(<<"Custom-Channel-Vars">>, JObj, wh_json:new())
          ),
+
+    EndpointFilter = fun(Element) ->
+        case Element of
+            {<<"Outbound-Caller-ID-Number">>, Number} -> false;
+            {<<"Outbound-Caller-ID-Name">>, Name}     -> false;
+            _OtherValues                              -> true
+        end
+    end,
+
+    FmtEndpoints = format_endpoints(Endpoints, Number, JObj, AccountId, EndpointFilter),
+
```

Comment:
Yes but then you have to pass those values around, and if we were to add more filters in the future, the values used there would then also have to be passed.  As opposed to just passing the fun.